imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -369,7 +369,6 @@ function cartToSalesOrder(cart) {
   const salesOrder = {
     companyCode: companyCode,
     type: "SalesOrder",
-    code: cart._id,
```

Comment:
Just to confirm: So when we call `estimateCart` which requires `salesOrder` object... no form of Id/code is needed on it?